src/components/Select/InternalComponents/MultiSelectSearchable.tsx

```diff
@@ -43,6 +43,10 @@ export const MultiSelectSearchable = () => {
 		getToggleButtonProps();
 	const { id, onKeyDown, ...restInputProps } = getInputProps();
```

Comment:
```suggestion
	const { id, onKeyDown, ...restInputProps } = getInputProps({ onFocus: undefined });
```
This works for me on my local. It preserves the keyboard behavior.  Please verify.

Reply:
Unfortunately, this can not be reproduced.